.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust: [stable, beta, nightly]
+
+    env:
+      RUSTFLAGS: -D warnings
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        rustup set profile minimal
+        rustup install ${{ matrix.rust }}
+        rustup default ${{ matrix.rust }}
+
+        cargo build --features v2_6_32
+        cargo build --features v3_2
+        cargo build --features v3_10
+        cargo build --features v4_2
+        cargo build --features v4_4
+        cargo build --features v4_20
+        cargo build --features v5_4
+        cargo build --features v5_11
+        cargo build --features "v3_2 netlink"
+        cargo build --no-default-features --features "v5_11 netlink"
+
+  gen:
+    name: Update generated files
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+        sudo apt install libclang-dev
+        cd gen && cargo run
```

Comment:
I've found it's worth adding --release here as bindgen runs significantly faster that way.

Reply:
Added. Will report back if is saves time.

Reply:
5m 11s after compared to 5m 59s before. I think `--release` is indeed worthwhile.